molgenis/molgenis-emx2
# fix: allow setting of inherited ref table columns. closes #3670

closes #3670

What are the main changes you did:
- fix #3670 

how to test:
1: 
- See issue

2: 
- Go to the schema editor (of the test schema in the issue)
- Go to Rasopathies
- Edit diabetesTreatment
- See that you can select a ref-column (before the fix it said 'No options provided'

todo:
